client/src/components/Projects/ColumnHeaderPopups/TextPopup.js

```diff
@@ -67,15 +81,33 @@ const TextPopup = ({
         />
         <hr style={{ width: "100%" }} />
       </div>
-      <input
+      {/*  <div>
+        <ul>
+          {textAllCurrentProjects.map((text, index) => (
+            <li key={index}>{text}</li>
+          ))}
+        </ul>
+      </div> */}
+      {/* <input
         type="text"
         placeholder="Search by Partial Text"
         onChange={e => {
           setNewSearchString(e.target.value);
         }}
         value={newSearchString}
-      />
-
+      /> */}
+      <ReactSelect
+        options={selectOptions.map(text => ({
```

Comment:
I think the list would be more user-friendly if it was sorted alphabetically.

Reply:
This is completed.